### PR TITLE
Render openshift-cluster-kube-scheduler-operator assets

### PR DESCRIPTION
**Pull request by aveshagarwal**
@deads2k  @sjenning @derekwaynecarr

### CODE REVIEW

**Comment by aveshagarwal** [comment]
@abhinavdahiya updated PTAL.

File: pkg/asset/ignition/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -200,17 +201,18 @@ func (a *bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
...
+		CloudProvider:              getCloudProvider(installConfig),
+		CloudProviderConfig:        getCloudProviderConfig(installConfig),
+		DebugConfig:                "",
+		KubeCoreRenderImage:        "quay.io/coreos/kube-core-renderer-dev:3b6952f5a1ba89bb32dd0630faddeaf2779c9a85",
+		EtcdCertSignerImage:        "quay.io/coreos/kube-etcd-signer-server:678cc8e6841e2121ebfdb6e2db568fce290b67d6",
+		EtcdctlImage:               "quay.io/coreos/etcd:v3.2.14",
+		BootkubeImage:              "quay.io/coreos/bootkube:v0.10.0",
+		ReleaseImage:               releaseImage,
+		HyperkubeImage:             "openshift/origin-node:latest",
+		KubeSchedulerOperatorImage: "docker.io/openshift/origin-cluster-kube-scheduler-operator:latest",
```
Comment (abhinavdahiya) [inline pkg/asset/ignition/bootstrap/bootstrap.go]: You can drop this.

**Comment by aveshagarwal** [comment]
@abhinavdahiya fixed as per your comment, PTAL.

**Comment by abhinavdahiya** [comment]
@aveshagarwal #322 was merged; can you rebase this.

**Comment by aveshagarwal** [comment]
> @aveshagarwal #322 was merged; can you rebase this.

done.

**Pull request merged**